pkg/server/healthProbes.go

```diff
@@ -10,13 +10,16 @@ import (
 )
 
 // LivenessProbe is used to check if this service is alive.
-func LivenessProbe(w http.ResponseWriter, r *http.Request) {
-	klog.V(7).Info("livenessProbe")
+func (s *ServerConfig) LivenessProbe(w http.ResponseWriter, r *http.Request) {
+	klog.Info("livenessProbe")
 	fmt.Fprint(w, "OK")
 }
 
 // ReadinessProbe checks if this service is available.
-func ReadinessProbe(w http.ResponseWriter, r *http.Request) {
-	klog.V(7).Info("readinessProbe")
+func (s *ServerConfig) ReadinessProbe(w http.ResponseWriter, r *http.Request) {
+	klog.Info("readinessProbe")
+	if !s.Dao.DBInitialized {
+		fmt.Fprint(w, "ReadinessProbe failed with error: Postgres db not initialized")
```

Comment:
1. I believe Kubernetes will prepend this part `ReadinessProbe failed with error:`
2. Maybe this is more accurate? `Unable to establish connection to Postgres.`
```suggestion
		fmt.Fprint(w, "Unable to establish connection to Postgres.")
```